docs/notebooks/sdba-advanced.ipynb

```diff
@@ -346,21 +346,18 @@
     "\n",
```

Comment:
<p>Assuming that original full period is a total of 30 years, so:</p><p>[01-01-2000, 01-01-2030[</p><p>i.e. excluding 2030, including 2029.</p><p> </p><pre class="ql-syntax" spellcheck="false">Window=15, stride=5

[01-01-2000, 01-01-2015[
[01-01-2005, 01-01-2020[
[01-01-2010, 01-01-2025[
[01-01-2015, 01-01-2030[
</pre><p>So in this case I don't see how you must drop one year, am I missing something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Ok. Then I agree, having 31 years will be awkward with the stacking. 

It seems this was intentional, previously, there was a selection afterwards to restrict `sim` to a 30 years period. Anyways, I think reminding that the dataset here "covers 31 years" and, maybe say that a dataset with 30 years is more natural with those parameters (unchanged by stacking/unstacking)